src/schema.mjs

```diff
@@ -172,10 +172,14 @@ export const arweave = {
         },
         uri: {
           type: "string",
-          pattern: "ar://[a-zA-Z0-9-_]{43}.*",
+          pattern: "ar:\/\/[a-zA-Z0-9-_]{43}.*"
         },
+        gateway: {
+          type: "string",
+          pattern: "^https?:\/\/[a-zA-Z0-9-.]{0,}(arweave)[a-zA-Z0-9-.]{0,}\/",
```

Comment:
instead, use the "format": "uri" option

Reply:
https://github.com/neume-network/schema/blob/main/src/schema.mjs#L344

Reply:
What do you mean?